scripts/postinstall.js

```diff
@@ -8,26 +8,41 @@ const isWindows = process.platform === 'win32';
 
 const truthyStr = (val) => val && !['0', 'false', 'f', 'n', 'no'].includes(val.toLowerCase());
 const { CI, ADBLOCK, SILENT } = process.env;
-if (!truthyStr(CI) && !truthyStr(ADBLOCK) && !truthyStr(SILENT)) {
-  const messageTokens = ['Serverless Framework successfully installed!'];
-  if (isStandaloneExecutable && !isWindows) {
-    messageTokens.push(
-      'To start your first project, please open another terminal and run “serverless”.',
-      'You can uninstall at anytime by running “serverless uninstall”.'
+const isNpmGlobalPackage = require('../lib/utils/npmPackage/isGlobal');
+const isNpmPackageWritable = require('../lib/utils/npmPackage/isWritable');
+
+(async () => {
+  if (!truthyStr(CI) && !truthyStr(ADBLOCK) && !truthyStr(SILENT)) {
+    const messageTokens = ['Serverless Framework successfully installed!'];
+
+    if (
```

Comment:
I wasn't sure about one thing - should we always show it or if someone already has `.serverlessrc` with that setting enabled we should skip this message? What do you think?

Reply:
Good question. I think I'll keep it, as this message is shown after _manual_ install. So even if user has _auto_ updates set, he may for some reason forgot about it, and that's a nice reminder